tox.ini

```diff
@@ -101,6 +101,7 @@ setenv=
     PYTEST_COLOR={tty:--color=yes}
     MYPY_COLOR={tty:--color-output}
 
+    PYTEST_ARGS=
```

Comment:
Can you please remove this? Thanks

Reply:
I'll remove the commit for this, but I added this as I was running into issues running tox locally for python <3.12. Anyway, I think #11838 can be done now.